engine/client/cl_main.c

```diff
@@ -1072,6 +1071,22 @@ static void CL_SendConnectPacket( void )
 	cls.timestart = Sys_DoubleTime();
 }
 
+/*
+=================
+CL_GetTestFragmentSize
+
+Returns bandwidth test fragment size
+=================
+*/
+static int CL_GetTestFragmentSize( void )
+{
+	const int fragmentSizes[CL_TEST_RETRIES] = { 64000, 32000, 10666, 5200, 1400 };
```

Comment:
Huh. Ideally we could save all CRCs for these values on server for quicker check. But that's something that could be done later.

Reply:
Also, instead of relying on gfx.wad file content, it's better to use pseudorandom sequence of bytes with "pre-shared" seed. This should be more reliable.